packages/shared/src/components/sidebar/ClickableNavItem.tsx

```diff
@@ -18,25 +24,24 @@ export function ClickableNavItem({
   children,
   ...props
 }: ClickableNavItemProps): ReactElement {
-  if (showLogin) {
-    return (
-      <button
-        {...props}
-        type="button"
-        className={classNames(navBtnClass, props.className)}
-        onClick={showLogin}
-      >
-        {children}
-      </button>
-    );
-  }
+  const onClick = useCallback(
+    (event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => {
+      if (showLogin) {
+        event.preventDefault();
+        showLogin();
+      } else {
+        item.action?.();
+      }
+    },
+    [showLogin, item],
+  );
 
   if (!isButton && (!item.action || item.path)) {
     return (
       <Link href={item.path} passHref prefetch={false}>
         <a
-          {...(item.action && { onClick: item.action })}
           {...props}
+          {...combinedClicks(onClick)}
```

Comment:
I'm not sure this needs combinedClicks though, you stay in the app anyway so maybe same page is fine for this?

Reply:
combinedClicks doesn't open anything in new tab for you, it just sends the input function to both onClick and onAuxClick.
It would be the same as doing
```ts
onClick={onClick}
onAuxClick={(event) => event.button === 1 && onClick}
```